kotlin/settings.gradle

```diff
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+rootProject.name = "workflow"
```

Comment:
What does this do?

Reply:
(At least) two things:
![image](https://user-images.githubusercontent.com/101754/52679833-34d3d180-2eeb-11e9-8818-4fbc4b1bfdf7.png)
![image](https://user-images.githubusercontent.com/101754/52679867-5a60db00-2eeb-11e9-8c7f-edd0a1a7e4e4.png)

Without that name, the project shows up as being called "kotlin" everywhere.